frontend/src/components/UserMainComponent.vue

```diff
@@ -8,95 +8,62 @@
-                        <tr>
-                          <td>
-                            <img src="https://avataaars.io/?avatarStyle=Circle&topType=LongHairDreads&accessoriesType=Kurt&hairColor=Auburn&facialHairType=BeardMedium&facialHairColor=Red&clotheType=BlazerShirt&clotheColor=Blue01&eyeType=Wink&eyebrowType=FlatNatural&mouthType=Twinkle&skinColor=Brown" width="70">
-                          </td>
-                          <td style="width: 3em">
-                          </td>
-                          <td style="text-align: left; width: 30em">
-                            <h4>Jon Doe</h4>
-                            <h5>john@gmail.com</h5>
-                          </td>
-                          <td>
-                            <button class="btn btn-primary">Follow</button>
-                          </td>
-                        </tr>
-                      </tbody>
-                    </table>
-                  </div>
-                </div>
-              </div>
-            </div>
-            <div class="card shadow mb-3" style="background: #1a202c">
-              <div class="card-body">
-                <div class="user-wrapper-container">
-                  <div class="user-wrapper">
-                    <table>
-                      <tbody>
-                        <tr>
-                          <td>
-                            <img src="https://avataaars.io/?avatarStyle=Circle&topType=LongHairDreads&accessoriesType=Kurt&hairColor=Auburn&facialHairType=BeardMedium&facialHairColor=Red&clotheType=BlazerShirt&clotheColor=Blue01&eyeType=Wink&eyebrowType=FlatNatural&mouthType=Twinkle&skinColor=Brown" width="70">
-                          </td>
-                          <td style="width: 3em">
-                          </td>
-                          <td style="text-align: left; width: 30em">
-                            <h4>John De</h4>
-                            <h5>john@gmail.com</h5>
-                          </td>
-                          <td>
-                            <button class="btn btn-primary">Follow</button>
-                          </td>
-                        </tr>
-                      </tbody>
-                    </table>
-                  </div>
-                </div>
-              </div>
-            </div>
+              <UserListComponent v-for="user in user_list" :user="user" :key="user.user_id" @show="show"/>
           </div>
         </div>
-      </div>
+      </div> 
     </div>
   </div>
 </template>
 
 <script>
+import axios from 'axios'
+import { ref } from 'vue';
+import UserListComponent from '../components/UserListComponent.vue'
+import commonService from '../composables/commonService'
+
 export default {
-  name: 'UserMainComponent'
+  name: 'UserMainComponent',
+  components: {
+    UserListComponent
+  },
+  setup() {
+    const csvc = commonService()
+    const user_list = ref()
+    const alert_button = ref(false)
+    const hide_alert = ref(false)
```

Comment:
Use camelCase format in javascript. If you see other parts of the code, please update as well

```suggestion
    const userList = ref()
    const alertButton = ref(false)
    const hideAlert = ref(false)
```